samples/asynctests/test_azure_event_hubs_receive_async.py

```diff
@@ -129,6 +128,35 @@ def on_message_received(message):
             log.info("Sequence Number: {}".format(annotations.get(b'x-opt-sequence-number')))
 
 
+@pytest.mark.asyncio
+async def test_event_hubs_receive_with_runtime_metric_async(live_eventhub_config):
+    uri = "sb://{}/{}".format(live_eventhub_config['hostname'], live_eventhub_config['event_hub'])
+    sas_auth = authentication.SASTokenAsync.from_shared_access_key(
+        uri, live_eventhub_config['key_name'], live_eventhub_config['access_key'])
+    source = "amqps://{}/{}/ConsumerGroups/{}/Partitions/{}".format(
+        live_eventhub_config['hostname'],
+        live_eventhub_config['event_hub'],
+        live_eventhub_config['consumer_group'],
+        live_eventhub_config['partition'])
+
+    receiver_runtime_metric_symbol = b'com.microsoft:enable-receiver-runtime-metric'
+    symbol_array = [types.AMQPSymbol(receiver_runtime_metric_symbol)]
+    desired_capabilities = utils.data_factory(types.AMQPArray(symbol_array))
+
+    async with uamqp.ReceiveClientAsync(source, debug=False, auth=sas_auth, timeout=1000, prefetch=10,
+                                        desired_capabilities=desired_capabilities) as receive_client:
+        message_batch = await receive_client.receive_message_batch_async(10)
+        log.info("got batch: {}".format(len(message_batch)))
+        for message in message_batch:
+            annotations = message.annotations
+            delivery_annotations = message.delivery_annotations
+            log.info("Sequence Number: {}".format(annotations.get(b'x-opt-sequence-number')))
+            log.info("Last enqueued sequence number: {}".format(delivery_annotations.get(b'last_enqueued_sequence_number')))
+            log.info("Last enqueued offset: {}".format(delivery_annotations.get(b'last_enqueued_offset')))
+            log.info("Last enqueued time utc: {}".format(delivery_annotations.get(b'last_enqueued_time_utc')))
+            log.info("Runtime info retrieval time utc: {}".format(delivery_annotations.get(b'runtime_info_retrieval_time_utc')))
```

Comment:
These log statements should be assert statements:
`assert b'last_enqueued_sequence_number' in annotations`

Reply:
test updated